string/trimRight.js

```diff
@@ -0,0 +1,19 @@
+/**
+Removes trailing chars from the right end of a string.
+
+@module string/trimRight
+@param {string} str The string to remove trailing chars from.
+@param {string} [chars=' '] The chars to remove.
+@returns {string} The given string without trailing chars.
+@example
+var trimRight = require('lildash/string/trimRight');
+trimRight('foo  ');  // => 'foo'
+trimRight('_foo_', '_');  // => '_foo'
+trimRight('!--foo--!', '!-');  // => '!--foo'
+*/
+module.exports = function (str, chars) {
+  chars = chars || ' ';
+  var pattern = new RegExp('([' + chars + ']*$)', 'g');
```

Comment:
Are you sure you want to deviate from the familiar _lodash_ API? 

If yes, I assume you mean for the default pattern to ultimately* equal _`/\s*$/g`_?

Should I take the given `pattern` and suffix it with _`*$`_ or should the user specify the entire pattern?

(If you can write a couple of test cases that should pass, then that would likely make things very clear for me. Thank you.)